doc/api/n-api.md

```diff
@@ -42,6 +42,39 @@ for the N-API C based functions exported by Node.js. These wrappers are not
 part of N-API, nor will they be maintained as part of Node.js. One such
 example is: [node-addon-api](https://github.com/nodejs/node-addon-api).
 
+## Implications of ABI Stability
+
+Although N-API provides an ABI stability guarantee, other parts of Node.js do
+not, and any external libraries used from the addon may not. In particular,
+neither of the following Node.js APIs provides an ABI stability guarantee:
```

Comment:
I think this wording is a bit misleading – we *do* have ABI stability guarantees, but we follow semver rather than not allowing *any* breakage.

Reply:
Good point - sorry!

Reply:
How about instead:

> In particular, none of the following APIs provide an ABI stability guarantee: